utils/utils.go

```diff
@@ -114,7 +115,7 @@ func FindNodesWithoutDeserializing(node *yaml.Node, jsonPath string) ([]*yaml.No
 func FindNodesWithoutDeserializingWithTimeout(node *yaml.Node, jsonPath string, timeout time.Duration) ([]*yaml.Node, error) {
 	jsonPath = FixContext(jsonPath)
 
-	path, err := jsonpath.NewPath(jsonPath)
+	path, err := jsonpath.NewPath(jsonPath, jsonpathconfig.WithPropertyNameExtension())
```

Comment:
I know there are more instances of this in the lib, would you be able to make sure we use this everywhere?

Reply:
Ah yes you are right. I think I got them all, other than the tests, which don't seem to need it anyway